q2_taxa/assets/index.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+
```

Comment:
Can you add a title here, too:

```jinja
{% block title %}q2-taxa barplot{% endblock %}
```

Maybe @gregcaporaso has something else he would like to see here instead?